src/App.vue

```diff
@@ -1,26 +1,25 @@
 <template>
   <div>
     <DeviceDetector :breakpoints="breakpoints" />
-    <BaseIdModalOpen class="x-start" modalId="x-components-app">Start</BaseIdModalOpen>
+    <BaseEventsModalOpen class="x-start">Start</BaseEventsModalOpen>
 
-    <BaseIdModal
-      @click.stop.native
+    <BaseEventsModal
+      :eventsToOpenModal="['UserClickedOpenEventsModal', 'UserOpenXProgrammatically']"
```

Comment:
With the `BaseEventsModal` I guess that there are no conflicts with other `BaseEventsModal` as you specify which events are allowed to open the modal.